src/static/css/cf-enhancements.less

```diff
@@ -1116,6 +1116,94 @@ textarea.input__long {
 
 
 /* topdoc
+  name: Base styles for definition lists
+  family: cf-core
+  notes:
+    - "Sets the term and definition in-line to each other but broken into rows"
+  tags:
+    - cf-core
+*/
+dt {
+    .h5();
+
+    display: inline-block;
```

Comment:
So we use `u-inline-block();` for IE hacks in several places. However, I've realized it's only for IE7 and below. We should probably either consistently still use it or remove it project-wide.